panel/dist/css/button.css

```diff
@@ -157,3 +157,12 @@
 :host(.bk-panel-models-widgets-FileDownload) .bk-btn .bk-TablerIcon {
   margin-left: 0.5em;
 }
+
+input[type='file'].bk-input {
+  padding-left: 10px;
+}
+
+input[type='file']::file-selector-button {
```

Comment:
How does this selector work?

Reply:
I found it by googling, and it seemed to do the trick :upside_down_face:  

https://developer.mozilla.org/en-US/docs/Web/CSS/::file-selector-button

Reply:
Ah, makes sense!